user/transmission/template.py

```diff
@@ -0,0 +1,74 @@
+pkgname = "transmission"
+pkgver = "4.0.6"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DENABLE_DAEMON=ON",
+    "-DENABLE_GTK=ON",
+    "-DENABLE_QT=ON",
+    "-DENABLE_CLI=ON",
+    "-DINSTALL_LIB=ON",
+]
+hostmakedepends = ["cmake", "gettext", "libxml2-progs", "ninja", "pkgconf"]
+makedepends = [
+    "gtkmm-devel",
+    "libcurl-devel",
+    "libdeflate-devel",
+    "linux-headers",
+    "miniupnpc-devel",
+    "qt6-qtbase-devel",
+    "qt6-qtsvg-devel",
+    "qt6-qttools-devel",
+]
+pkgdesc = "BitTorrent client"
+maintainer = "tulilirockz <tulilirockz@outlook.com>"
+license = "GPL-2.0-or-later"
+url = "https://github.com/transmission/transmission"
+source = f"{url}/releases/download/{pkgver}/transmission-{pkgver}.tar.xz"
+sha256 = "2a38fe6d8a23991680b691c277a335f8875bdeca2b97c6b26b598bc9c7b0c45f"
+
+
+def post_install(self):
+    self.install_service(self.files_path / "transmission-daemon")
+
+
+@subpackage("transmission-qt")
+def _(self):
+    self.depends = [self.parent]
+    return [
+        "usr/bin/transmission-qt",
+        "usr/share/man/man1/transmission-qt.1",
+        "usr/share/applications/transmission-qt.desktop",
+        "usr/share/transmission/translations/*",
+    ]
+
+
+@subpackage("transmission-gtk")
+def _(self):
+    self.depends = [self.parent]
+    return [
+        "usr/bin/transmission-gtk",
```

Comment:
```suggestion
        "cmd:transmission-gtk",
```

and remove the .1, same as below